src/hal/src/command/raw.rs

```diff
@@ -243,14 +263,14 @@ pub trait RawCommandBuffer<B: Backend>: Clone + Any + Send + Sync {
         T: IntoIterator,
         T::Item: Borrow<Rect>;
 
-    ///
+    /// DOC TODO: I don't know how to explain this.
```

Comment:
Sets the stencil reference value for comparison operations and store operations.
Will be used on the LHS for stencil compare ops and as store value when the store op is `Reference`.